content/modules/redisearch/redisearch-2-upgrade.md

```diff
@@ -0,0 +1,61 @@
+---
+Title: Upgrading a Database to RediSearch 2.x
+description:
+weight: 30
+alwaysopen: false
+categories: ["Modules"]
+module: RediSearch
+---
+RediSearch 2.x includes some significant changes to the architecture of RediSearch that provide improved functionality.
+The main architectural change is that the indexes are stored outside of the redis database that contains the data.
+This allows for improvements in command efficiency.
+It also allows for improvements in replication between clusters because the index changes are managed by the participating clusters rather than being synchronized with the data.
+
+This change allows databases with RediSearch to support:
+
+- Active-Active databases
+- Database clustering (sharding)
+- Replica Of to sharded destination database
+- EXPIRE of documents reflected in the index
+
+To upgrade a Redis Enterprise Software (RS) database with RediSearch 1.x to RediSearch 2.x, you have to set up a new database with RediSearch 2.x and use the `RediSearch_Syncer.py` script to replicate the data from the old database into the new database.
```

Comment:
How will we distribute this script @MeirShpilraien

Reply:
Ok found the s3 buckets, suggest we add the links to the packages here and not have it in our download centre